src/unsplash.ts

```diff
@@ -6,12 +6,15 @@ export async function getRandomUnsplashImageUrl(
   width: number,
   height: number
 ): Promise<string> {
-  // localhost:3000/?width=200&height=300
   const randomImageUrl = `${globalConfig.unsplash.apiBaseUrl}/photos/random?client_id=${globalConfig.unsplash.apiAccessKey}`;
   const { data } = await axios.get(randomImageUrl);
   let url = data.urls.full;
 
-  const imageUrl = createUrlWithParams(url, { width, height });
+  let params: any = {};
+  if (!isNaN(width)) params.w = width;
+  if (!isNaN(height)) params.h = height;
+
+  const imageUrl = createUrlWithParams(url, params);
```

Comment:
This logic has been used twice. It's good to extract the logic into separate function.